package.json

```diff
@@ -2,48 +2,58 @@
-  "main": "snabbdom.js",
-  "module": "es/snabbdom.js",
-  "typings": "snabbdom.d.ts",
+  "type": "module",
+  "main": "./es/snabbdom.js",
+  "exports": {
+    ".": "./es/snabbdom.js",
+    "./h": "./es/h.js",
+    "./attributes": "./es/modules/attributes.js",
+    "./class": "./es/modules/class.js",
+    "./dataset": "./es/modules/dataset.js",
+    "./eventlisteners": "./es/modules/eventlisteners.js",
+    "./props": "./es/modules/props.js",
+    "./style": "./es/modules/style.js",
+    "./tovnode": "./es/tovnode.js"
+  },
+  "typings": "es/snabbdom.d.ts",
   "directories": {
     "example": "examples",
     "test": "test"
   },
   "devDependencies": {
-    "@commitlint/cli": "8.3.5",
-    "@commitlint/config-conventional": "8.3.4",
-    "@commitlint/travis-cli": "8.3.5",
-    "@types/assert": "1.4.6",
-    "@types/chai": "4.2.11",
-    "@types/lodash.shuffle": "4.2.6",
-    "@types/mocha": "7.0.2",
-    "@typescript-eslint/eslint-plugin": "2.29.0",
-    "benchmark": "2.1.4",
-    "browserify": "16.5.0",
-    "chai": "4.2.0",
-    "editorconfig-checker": "3.0.4",
-    "eslint": "6.8.0",
-    "eslint-config-standard-with-typescript": "16.0.0",
-    "eslint-plugin-import": "2.20.1",
-    "eslint-plugin-node": "11.0.0",
-    "eslint-plugin-promise": "4.2.1",
-    "eslint-plugin-standard": "4.0.1",
-    "husky": "4.2.3",
-    "karma": "4.4.1",
-    "karma-browserstack-launcher": "1.5.1",
-    "karma-chrome-launcher": "3.1.0",
-    "karma-firefox-launcher": "1.3.0",
-    "karma-mocha": "1.3.0",
-    "karma-mocha-reporter": "2.2.5",
-    "karma-webpack": "4.0.2",
-    "lodash.shuffle": "4.2.0",
-    "mocha": "7.1.0",
-    "npm-run-all": "4.1.5",
-    "remark-cli": "7.0.1",
-    "remark-toc": "7.0.0",
-    "tsconfigs": "4.0.2",
-    "typescript": "3.8.3",
-    "webpack": "4.42.0"
+    "@commitlint/cli": "^8.3.5",
+    "@commitlint/config-conventional": "^8.3.4",
+    "@commitlint/travis-cli": "^8.3.5",
+    "@types/assert": "^1.4.6",
+    "@types/chai": "^4.2.11",
+    "@types/lodash.shuffle": "^4.2.6",
+    "@types/mocha": "^7.0.2",
+    "@typescript-eslint/eslint-plugin": "^2.30.0",
+    "benchmark": "^2.1.4",
+    "chai": "^4.2.0",
+    "editorconfig-checker": "^3.0.4",
+    "eslint": "^6.8.0",
+    "eslint-config-standard-with-typescript": "^16.0.0",
+    "eslint-plugin-import": "^2.20.2",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^4.0.1",
+    "husky": "^4.2.5",
+    "karma": "^4.4.1",
+    "karma-browserstack-launcher": "^1.5.2",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-firefox-launcher": "^1.3.0",
+    "karma-mocha": "^1.3.0",
+    "karma-mocha-reporter": "^2.2.5",
+    "karma-webpack": "^4.0.2",
+    "lodash.shuffle": "^4.2.0",
+    "mocha": "^7.1.2",
+    "npm-run-all": "^4.1.5",
+    "remark-cli": "^7.0.1",
+    "remark-toc": "^7.0.0",
+    "tsconfigs": "^4.0.2",
+    "typescript": "^3.8.3",
+    "webpack": "^4.43.0"
```

Comment:
1. The versions are pinned on purpose. We use Renovate to keep them up-to-date. In theory. In practice, #595 is making this tedious.
1. If you bump any packages, please explain why that is necessary, and please bump only those packages.